### DISCUSSION TITLE
Logs out on clicking "identities" and "datastore" menu

### DISCUSSION THREAD

**Discussion by yashpatil1998** [OpenIdentityPlatform/OpenAM#574] [Q&A] [answered]
Hi community,

I have followed the steps given [here](https://www.openidentityplatform.org/blog/openam-opendj-docker) to run openam with opendj in a docker-compose. 
I want to add identities for which I should go to a realm -> identities. 
When i click identities, i automatically get logged out. 
Since i'm running a docker image, i'm assuming my system's installed jdk shouldn’t be of a concern. 
Requesting help on my problem.

**Comment by maximthomas** [comment, marked as answer]
@yashpatil1998 hi, seems there was an issue with the legacy UI and JDK 17 in OpenAM 14.7.0 (#570). As a workaround you can temporary downgrade and use OpenAM 14.6.6 Docker image. We will release a patch as soon as possible.

**Reply by maximthomas** [reply]
@yashpatil1998 You can also use openidentityplatform/openam:14.7.0-alpine image

**Reply by yashpatil1998** [reply]
Thanks Maxim, the alpine image worked for me.

**Reply by maximthomas** [reply]
Now you can also use openidentityplatform/openam:14.7.1 image
